src/Controller/Game/FleetController.php

```diff
@@ -86,46 +77,51 @@ public function recall(Request $request, int $fleetId): Response
      */
     public function reinforce(Request $request, int $fleetId): Response
     {
-        $fleet = $this->fleetRepository->find($fleetId);
+        $fleet = $this->getFleetByIdAndPlayer($fleetId, $this->getPlayer());
```

Comment:
Currently there is a coupling between a Repository and a Controller, although, what if you were to add 'commandline' controllers? You could consider abstracting all your business logic away to another layer that is testable without the need for HTTP Requests and Responses.